packages/web-api/src/api-contracts/scan-result-response.ts

```diff
@@ -20,6 +17,12 @@ export type ScanResultResponse =
       }
     | ScanResultErrorResponse;
 
+export interface ScanResultErrorResponse {
+    scanId: string;
+    code: WebApiErrorCodeName;
+    error: string;
+}
```

Comment:
we may need to send {
url,
code, 
error
}
& not scan id if there is error

Reply:
The ScanRunResponse is type when we have error before we assign scan id. Other cases will have scan id already so we can use above type.